src/components/button/button.tsx

```diff
@@ -94,6 +94,8 @@ export class Button {
                         outlined: this.outlined,
                     }}
                     disabled={this.disabled || this.loading}
+                    aria-busy={this.loading ? 'true' : 'false'}
```

Comment:
If I am allowed to suggest, here is my suggestion. I think we can simplify it by removing the ternary expression. What do you think?

```suggestion
                    aria-busy={String(this.loading)}
```

Reply:
I think using `String(this.loading)` makes this code harder to read than using the ternary. But that's a matter of personal preference of course.

But, do we even have to translate the boolean to a string?

Reply:
> But, do we even have to translate the boolean to a string?

I don't know if we have to translate it to string, maybe not. `this.loading` might be enough then

Reply:
I've already approved this 
```tsx
aria-busy={this.loading ? 'true' : 'false'}
```
in an earlier PR on info-tile: https://github.com/Lundalogik/lime-elements/pull/3185/files


I think this is better for screen readers, to explicitly set aria-busy="false" when the loading state is complete. This ensures that the element consistently communicates its status to assistive technologies, avoiding any confusion.

if we go for "code simplicity" and write:
```tsx
aria-busy={this.loading}
```
the `aria-busy` attribute will disappear when loading is false, which could lead to screen readers not detecting the end of the loading state as clearly as when it's explicitly set to false. This would make the experience more predictable and accessible.

Reply:
Yeah, no, I didn't mean we should prioritise code simplicity over the quality of the _output_. I only meant if the result was the same 😅